client/my-sites/site-settings/reading-rss-feed-settings/index.tsx

```diff
@@ -12,7 +12,7 @@ type Fields = {
 type RssFeedSettingsSectionProps = {
 	fields: Fields;
 	onChangeField: ( field: string ) => ( event: React.ChangeEvent< HTMLInputElement > ) => void;
-	handleSubmitForm: ( event: React.FormEvent< HTMLFormElement > ) => void;
+	handleSubmitForm: ( event?: React.FormEvent< HTMLFormElement > ) => void;
```

Comment:
This change (and similar ones in this PR) is needed since `handleSubmitForm` is used as `onButtonClick` handler in the `SettingsSectionHeader` component which is actually not sending a form so we don't have such event available.